### PR TITLE
[G2M] switch, range-label & date-range components new color palette classname

### PR DESCRIPTION
**Pull request by kc-leung**
- changed switch component color classname to the new added color palette
- changed range-slider component color classname to the new added color palette
- changed date-range component color classname to the new added color palette
- changed switch base component custom classes logic

### CODE REVIEW

File: src/base-components/range-slider-base/index.js [hunk 1]
```diff
@@ -8,15 +8,15 @@ import './range-slider-base.css'
...
+  const thumbColor = getTailwindConfigColor(`${classes.thumbColor}-500`)
 
   const sliderClasses = Object.freeze({
     sliderContainer: `${width} relative my-2.5 h-px`,
     thumb: `${width}`,
     thumbColor: thumbColor ? thumbColor : '#000',
     slider: 'h-1 rounded-sm',
-    sliderTrack: `${classes.sliderTrack ? classes.sliderTrack : 'bg-gray-300'}`,
-    sliderRange: `${classes.sliderRange ? classes.sliderRange : 'bg-black'}`,
+    sliderTrack: `${classes.sliderTrack ? `bg-${classes.sliderTrack}-200` : 'bg-secondary-400'}`,
```
Comment (hyx131) [inline src/base-components/range-slider-base/index.js:18]: same comment as https://github.com/EQWorks/lumen-labs/pull/21#discussion_r673276823

**Review by woozyking — APPROVED** [review]
For switches, shall we also consider accessibility measures such as including stripes for colour-weak/blind users?

**Review by YizhiCatherineZhang — CHANGES_REQUESTED** [review]
Since this is built with Tailwind CSS and the color guide is there. I suggest you change the color based on our color guide line to keep the consistency. 

Function wise are good, and we can change the style later.

**Comment by woozyking** [comment]
this might just be me but the rectangle inner span seems to be positioned a little toward top. minor though

pos

@kc-leung

**Pull request merged**